src/Image/Image.tsx

```diff
@@ -123,7 +114,7 @@ export class Image extends React.Component<
             {PlaceholderContent}
```

Comment:
Shouldn't work as a child of `View`

Reply:
`PlaceholderContent` is of type `ReactNode` so user have to use `Text` Component for text, not directly

Reply:
PlaceholderContent type is `ReactElement` so user can do `PlaceholderContent={<Text>Hey</Text>}`